app/models/manageiq/providers/vmware/infra_manager/vm/reconfigure.rb

```diff
@@ -345,6 +357,40 @@ def remove_network_adapter_config_spec(vim_obj, vmcs, options)
     end
   end
 
+  def connect_cdrom_config_spec(vim_obj, vmcs, hardware, cdrom)
+    device = vim_obj.send(:getDeviceByLabel, cdrom[:device_name], hardware)
```

Comment:
I know we did this before this change, but why do we `send` here. Are these methods private? If so, should they be public?

Reply:
Hm good question I don't know why we do that, let me try just calling it normally.

Reply:
Works fine, I'll change these over